packages/apps/human-app/server/src/modules/jobs-discovery/jobs-discovery.controller.ts

```diff
@@ -0,0 +1,40 @@
+import { Controller, Get, Headers, Query } from '@nestjs/common';
+import { ApiOperation, ApiTags } from '@nestjs/swagger';
+import { InjectMapper } from '@automapper/nestjs';
+import { Mapper } from '@automapper/core';
+import { JobsDiscoveryService } from './jobs-discovery.service';
+import {
+  JobsDiscoveryParamsCommand,
+  JobsDiscoveryParamsDto,
+  JobsDiscoveryResponse,
+} from './interfaces/jobs-discovery.interface';
+
+@Controller()
+export class JobsDiscoveryController {
+  constructor(
+    private readonly jobsDiscoveryService: JobsDiscoveryService,
+    @InjectMapper() private readonly mapper: Mapper,
+  ) {}
+
+  @ApiTags('Jobs-Discovery')
+  @Get('/jobs')
+  @ApiOperation({
+    summary:
+      'Retrieve a list of filtered available jobs for passed Exchange Oracle url',
+  })
+  public async getJobs(
+    @Query() jobsDiscoveryParamsDto: JobsDiscoveryParamsDto,
+    @Headers('authorization') token: string,
```

Comment:
Can you use guards for the cases that require auth header, so that we can see this in swagger:
![image](https://github.com/humanprotocol/human-protocol/assets/61605646/f77e0fff-249b-4f9f-8ba8-8ef9c6239e7d)
I know this header is passed to ex oracle for auth, but can just don't validate the jwt and forward it to the Ex oracle